accounts/external/backend.go

```diff
@@ -131,6 +131,12 @@ func (api *ExternalSigner) Accounts() []accounts.Account {
 func (api *ExternalSigner) Contains(account accounts.Account) bool {
 	api.cacheMu.RLock()
 	defer api.cacheMu.RUnlock()
+	if api.cache == nil {
```

Comment:
Why not call `api.Accounts()` at the start of the function, before the lock is taken in the first place?